src/usr/local/www/services_dyndns_edit.php

```diff
@@ -323,12 +323,12 @@ function build_if_list() {
 
 $section->addInput(new Form_Checkbox(
 	'proxied',
-	'CloudFlare Proxy',
+	'Cloudflare Proxy',
 	'Enable Proxy',
 	$pconfig['proxied']
-))->setHelp('Note: This enables CloudFlares Virtual DNS proxy.  When Enabled it will route all traffic '.
+))->setHelp('Note: This enables Cloudflares Virtual DNS proxy.  When Enabled it will route all traffic '.
```

Comment:
This should probably be "... enables the Cloudflare Virtual DNS proxy", to avoid the awkward use of their name in a possessive.